extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -397,6 +412,12 @@ export class SchemaCompareDialog {
 			};
 		});
 
+		if (idx >= 0) {
```

Comment:
nit: Might as well make this >= 1 since if it's 0 you don't need to move it regardless